algoliasearch-common/src/main/java/com/algolia/search/APIClient.java

```diff
@@ -232,6 +232,24 @@ public Task copyIndex(
     return copyIndex(srcIndexName, dstIndexName, scopes, RequestOptions.empty);
   }
 
+  public Task deleteIndex(@Nonnull String indexName) throws AlgoliaException {
```

Comment:
Could you add the javadoc for those 2 methods?

Reply:
Yes right, totally forgot. Will do.